pkgs/development/libraries/science/math/suitesparse/default.nix

```diff
@@ -60,6 +60,12 @@ stdenv.mkDerivation rec {
   ]
   ;
 
+  env = lib.optionalAttrs stdenv.isDarwin {
+    # Ensure that there is enough space for the `fixDarwinDylibNames` hook to
+    # update the install names of the output dylibs.
+    NIX_CFLAGS_LINK = "-Wl,-headerpad_max_install_names";
```

Comment:
Does this work?

```suggestion
    NIX_LDFLAGS = "-headerpad_max_install_names";
```

Reply:
That works. I updated the commit and re-pushed.